File: PROGRAMMERS/lv0/javascript/배열 자르기.js
function solution(numbers, num1, num2) {
  return numbers.slice(num1, num2 + 1);
}

let inps = [
  [[1, 2, 3, 4, 5], 1, 3],
  [[1, 3, 5], 1, 2],
];

for (let i of inps) {
  console.log(solution(i[0], i[1], i[2]));
}

File: PROGRAMMERS/lv0/python/배열 자르기.py
def solution(numbers, num1, num2):
    return numbers[num1:num2+1]

inps = [
    [[1, 2, 3, 4, 5], 1, 3],
    [[1, 3, 5], 1, 2]
]

for (ns, n1, n2) in inps:
    print(solution(ns, n1, n2))